locator/abstract_replication_strategy.cc

```diff
@@ -68,6 +68,17 @@ inet_address_vector_replica_set abstract_replication_strategy::get_natural_endpo
     return res->second;
 }
 
+stop_iteration abstract_replication_strategy::for_each_natural_endpoint_until(const token& search_token, const effective_replication_map& erm, const noncopyable_function<stop_iteration(const inet_address&)>& func) const {
+    const token& key_token = erm.get_token_metadata_ptr()->first_token(search_token);
+    auto res = erm.get_replication_map().find(key_token);
+    for (const auto& ep : res->second) {
+        if (func(ep)) {
```

Comment:
stop_iteration is a bool_class so it is convertible to `bool`.
(Also for constructing as `stop_iteration(bool)`).

But I don't mind making this change if you think it makes the code clearer.

Reply:
Yes, I think even if this compiles it's very strange to return "yes" because we believe "return true" will not be clear enough, and one line up we think that if() will be clear enough.